TFLite/src/main.py

```diff
@@ -1,9 +1,14 @@
 import TFLiteParser
 import os
+from common import Storage
+import argparse
+
+import timeit
 
```

Comment:
Let's keep two section of imports. Standard python library imports(os, argparse, timeit) and third-party imports like TFLiteParser etc). All of them should be lexicographically sorted.

Reply:
Fixed imports

Reply:
Looks like all your updates are gone after the PR refactoring.